src/content/docs/apm/apm-ui-pages/monitoring/find-performance-problems-apm-transactions-page.mdx

```diff
@@ -0,0 +1,374 @@
+---
+title: 'Find performance problems using the APM Transactions page'
+tags:
+  - APM
+  - APM UI pages
+  - Monitoring
+translate:
+  - jp
+metaDescription: APM's Transactions page shows app, browser, and other transaction details, average response time, and throughput.
+redirects:
+  - /docs/applications-menu/web-transactions
+  - /docs/applications-menu/background-tasks
+  - /docs/applications-menu/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-dashboard
+  - /docs/apm/applications-menu/monitoring/transactions-page
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-performance-problems
+  - /docs/apm/applications-menu/monitoring/transactions-page-find-specific-performance-problems
+  - /docs/apm/transactions/intro-transactions/transactions-page-find-specific-performance-problems
+  - /docs/apm/apm-ui-pages/monitoring/transactions-page-find-specific-performance-problems
+---
+
+The APM **Transactions** page helps you identify transactions to look at more closely in order to troubleshoot performance problems and avoid errors. 
+
+In general, the longer a given transaction takes, and the more time it takes compared to other transactions, the more likely it is that it's a source of issues for a given app. Similarly, low throughput can be an indicator that not everything is working as it should. 
+
+We recommend you make a habit of reviewing app performance data on the APM **Transactions** page more closely in order to scan for new issues and diagnose the cause of problems you're already aware of.
+
+<Callout variant="tip">
+  At New Relic, a transaction is defined as one logical unit of work in a software application. For more information about transactions in the context of APM, see [Transactions in New Relic's APM](/docs/apm/transactions/intro-transactions/transactions-new-relic-apm/). 
+</Callout>
+
+## Transactions page overview
+
+Access the APM **Transactions** page by going to **[one.newrelic.com](https://one.newrelic.com) > APM > (select an app) > Monitor > Transactions**. 
+
+![Screenshot showing ransactions page default view](./images/APM-transactions-page-default-view.jpg "Transactions page default view")
+
+Find key performance metrics in the overview bar at the top of the **Transactions**. In the main chart, view the top five transactions by wall-clock time, visually arranged  according to how much time they took to complete. There's also clickable list of the top 20 transactions, a throughput graph, and a table of transaction traces. 
+
+### Wall clock time [#wall_clock_time]
+Wall-clock time is the amount of time recorded by the clock. For APM transactions data, New Relic uses wall-clock time for all transactions and then calculates that value across all transactions.
```

Comment:
"Wall-clock time is the amount of time recorded by the clock_, from when a request is received until it returns a response. This is different than CPU time, for example._"

For the second sentence: "New Relic calculates the sum of wall-clock time for all requests to a transaction, across all hosts and service instances. This sum is then broken down by transaction, to show how time is comparatively spent in each transaction."